config/config.go

```diff
@@ -6,27 +6,40 @@ import (
 	"github.com/kelseyhightower/envconfig"
 )
 
+// KafkaTLSProtocolFlag informs service to use TLS protocol for kafka
+const KafkaTLSProtocolFlag = "TLS"
+
 // Config represents service configuration for dp-import-cantabular-dataset
 type Config struct {
 	BindAddr                     string        `envconfig:"BIND_ADDR"`
 	GracefulShutdownTimeout      time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval          time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout   time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
-	KafkaAddr                    []string      `envconfig:"KAFKA_ADDR"                     json:"-"`
-	KafkaVersion                 string        `envconfig:"KAFKA_VERSION"`
-	KafkaOffsetOldest            bool          `envconfig:"KAFKA_OFFSET_OLDEST"`
-	KafkaNumWorkers              int           `envconfig:"KAFKA_NUM_WORKERS"`
-	KafkaMaxBytes                int           `envconfig:"KAFKA_MAX_BYTES"`
-	InstanceStartedGroup         string        `envconfig:"KAFKA_DATASET_INSTANCE_STARTED_GROUP"`
-	InstanceStartedTopic         string        `envconfig:"KAFKA_DATASET_INSTANCE_STARTED_TOPIC"`
-	CategoryDimensionImportTopic string        `envconfig:"CANTABULAR_DATASET_CATEGORY_DIMENSION_IMPORT"`
 	DatasetAPIURL                string        `envconfig:"DATASET_API_URL"`
 	RecipeAPIURL                 string        `envconfig:"RECIPE_API_URL"`
 	ImportAPIURL                 string        `envconfig:"IMPORT_API_URL"`
 	CantabularURL                string        `envconfig:"CANTABULAR_URL"`
 	CantabularHealthcheckEnabled bool          `envconfig:"CANTABULAR_HEALTHCHECK_ENABLED"`
 	ServiceAuthToken             string        `envconfig:"SERVICE_AUTH_TOKEN"         json:"-"`
 	ComponentTestUseLogFile      bool          `envconfig:"COMPONENT_TEST_USE_LOG_FILE"`
+	KafkaConfig                  KafkaConfig
```

Comment:
I wonder if this field could be name `kafka` rather than `kafkaConfig`?

Reply:
I'd say stick with KafkaConfig because there are other things that start with kafka in the containing structure, such as KafkaAddr, KafkaVersion and so on.

Reply:
KafkaConfig is in line with whats done in other ONS repo's along the lines of MongoConfig. So lead=ve as is.